### ISSUE TITLE
New haxe website

### ISSUE THREAD

**Issue by jasononeil** [dpeek/hvm#3]
It's worth noting that http://old.haxe.org/file/ wasn't updated with the new Haxe 3.2.0-rc.1 release.  I've copied the files there for now, but we probably need a solution for the next releases.

Files on the new site are:

http://haxe.org/website-content/downloads/
http://haxe.org/website-content/downloads/versions.json
http://haxe.org/website-content/downloads/3,1,3/downloads/

etc.  A few options:
- We make part of the release process to still put the files in the old location
- We get our script to use the JSON file above rather than scrape haxe.org (Note: the platform availability is missing from this file)
- I add some other kind of plain-text file to the website that we can use to scrape this information, and I could even make it nice and bash-friendly.

**Comment by dpeek** [comment]
Parsing that JSON seems easy enough, though we will need platform support info.

I'm actually OK with the current scraping (it's dirty but it works). The problem is having some downloads in the old place and others in the new – and the builds on nekovm.org are for the wrong arch on OS X. Ideally we could just get all the available versions on build.haxe.org (I've noticed one or two stable versions on there) and then haxe.org and hvm could just reference those directly? cc/ @waneck

I know 2.x support is an edge-case but it was my original motivation for all this (we've got some legacy projects at Massive)

**Comment by jasononeil** [comment]
Yes, I think all stable versions being on build.haxe.org, or else a page on haxe.org, and continue with the primitive scraping, sounds like the best approach.  If it's problematic for @waneck to put them on the build server I'll create a suitable page on the website.

**Comment by jasononeil** [comment]
Just wanted to note that the haxe.org website now has more standardised filenames for downloads going all the way back to 1.9.0

The formula is:

```
http://haxe.org/website-content/downloads/$semver/downloads/haxe-$semver-$targetSuffix
```

eg. http://haxe.org/website-content/downloads/3.1.3/downloads/haxe-3.1.3-win.exe

I know build.haxe.org is still preferable.  Just wanted to note it though.

**Comment by dpeek** [comment]
Is there somewhere we can get a list of available versions?

**Comment by elsassph** [comment]
@jasononeil how can we get the list of available versions?

**Comment by skial** [comment]
Theres a list of officially released versions at https://github.com/HaxeFoundation/haxe.org/blob/staging/downloads/versions.json
